exercises/react/react_test.py

```diff
@@ -121,19 +108,16 @@ def test_callbacks_can_be_added_and_removed(self):
         input_.value = 31
         self.assertEqual(cb1_observer[-1], 32)
         self.assertEqual(cb2_observer[-1], 32)
-
         output.remove_callback(callback1)
         output.add_callback(callback3)
         input_.value = 41
         self.assertEqual(cb2_observer[-1], 42)
         self.assertEqual(cb3_observer[-1], 42)
+        self.assertEqual(cb1_observer, [])
```

Comment:
`cb1_observer` is never cleared, so it shouldn't be expected to be empty.

Reply:
I am very confused with this specific test case, because I use `expect_callbacks_not_to_be_called` key to check if the observer is empty. I do not know how to solve the template to fix this error :/

the specification:
```    {
      "description": "callbacks can be added and removed",
      "property": "react",
      "input": {
        "cells": [
          {
            "name": "input",
            "type": "input",
            "initial_value": 11
          },
          {
            "name": "output",
            "type": "compute",
            "inputs": ["input"],
            "compute_function": "inputs[0] + 1"
          }
        ],
        "operations": [
          {
            "type": "add_callback",
            "cell": "output",
            "name": "callback1"
          },
          {
            "type": "add_callback",
            "cell": "output",
            "name": "callback2"
          },
          {
            "type": "set_value",
            "cell": "input",
            "value": 31,
            "expect_callbacks": {
              "callback1": 32,
              "callback2": 32
            }
          },
          {
            "type": "remove_callback",
            "cell": "output",
            "name": "callback1"
          },
          {
            "type": "add_callback",
            "cell": "output",
            "name": "callback3"
          },
          {
            "type": "set_value",
            "cell": "input",
            "value": 41,
            "expect_callbacks": {
              "callback2": 42,
              "callback3": 42
            },
            "expect_callbacks_not_to_be_called": ["callback1"]
          }
        ]
      },
      "expected": {}
    },
```

Reply:
See my next comment at exercises/react/.meta/template.j2:54

Reply:
I finally finished the code :)
But that has some hard coded :thinking: